esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -1970,6 +1970,38 @@ void CTpWrapper::getAttPath(const char* Path,StringBuffer& returnStr)
     JBASE64_Decode(Path, returnStr);
 }
 
+#ifdef _CONTAINERIZED
+void CTpWrapper::getContainerServices(double version, const char* serviceType, const char* serviceName, IArrayOf<IConstContainerService>& services)
```

Comment:
This is a different use of that same container word again.  Don't think it's right here either.  In my mind container services would imply the services in one particular container not the services for an entire hpcc cloud environment.